src/core/Table/TableRowMemoized.tsx

```diff
@@ -17,14 +18,18 @@ import { CSSTransition } from 'react-transition-group';
  */
 const TableRow = <T extends Record<string, unknown>>(props: {
   row: Row<T>;
-  rowProps: TableRowProps;
```

Comment:
What was `TableRowProps` giving before?

Reply:
key, classname, style and role but it only returns key and role